patches/api/0418-Piglin-Dancing-API.patch

```diff
@@ -0,0 +1,39 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Amin <amin.haddou@frg.wwschool.de>
+Date: Wed, 22 Feb 2023 10:27:19 +0100
+Subject: [PATCH] Piglin Dancing API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Piglin.java b/src/main/java/org/bukkit/entity/Piglin.java
+index d4cb4b0ed1d9766a87867dcf1a3a839526ba9332..44cbc24e1702161a17b21a151e945ab8b2b0a73f 100644
+--- a/src/main/java/org/bukkit/entity/Piglin.java
++++ b/src/main/java/org/bukkit/entity/Piglin.java
+@@ -109,6 +109,28 @@ public interface Piglin extends PiglinAbstract, InventoryHolder, com.destroystok
+      * @return is charging
+      */
+     boolean isChargingCrossbow();
++
++    /**
++     * Causes the piglin to dance
```

Comment:
I am bad with words, but this sounds wrong. setDancing(false) doesn't cause the piglin to dance, it stops them from dancing. idk how to word it better tho